CHANGES.md

```diff
@@ -1,3 +1,7 @@
+### v7.1.2 (2022-07-27)
+
+* TCP: fix memory leaks on connection close (#489 @TheLortex)
```

Comment:
In the issue comment you had quite a bit more information which scenarios were dealt with, it may be worth to be a bit more wordy in the changes entry as well...

Reply:
I have force-pushed with aac0e02f3b2e3da016f6f7cfb9790c4949be8456 containing more information about the various patches